### PR TITLE
Rework of dnd store

### PR DESCRIPTION
**Pull request by dmitryRomankov**

### CODE REVIEW

**Comment by dmitryRomankov** [comment]
[change orientation of drag-n-drop](https://trello.com/c/gMcbkYCS/49-change-orientation-of-drag-n-drop)

**Comment by dmitryRomankov** [comment]
[write methods for active cards fields(for my and enemy)](https://trello.com/c/mSssvggs/48-write-methods-for-active-cards-fieldsfor-my-and-enemy)

File: client/src/store/cards/cards.facade.ts [hunk 1]
```diff
@@ -1,22 +1,26 @@
 import { Injectable } from '@angular/core';
 import { Store } from '@ngrx/store';
 
+import { CdkDragDrop } from '@angular/cdk/drag-drop';
+
+import { Card } from 'models';
+
 import { CardsState } from './interfaces';
```
Comment (vharadkou) [inline client/src/store/cards/cards.facade.ts]: empty line

File: client/src/store/cards/cards.facade.ts [hunk 2]
```diff
@@ -1,22 +1,26 @@
 import { Injectable } from '@angular/core';
 import { Store } from '@ngrx/store';
```
Comment (vharadkou) [inline client/src/store/cards/cards.facade.ts]: empty line

File: client/src/store/cards/cards.reducer.ts [hunk 3]
```diff
@@ -58,6 +58,18 @@ export const cardsReducer = (
         ...state
       };
 
+    case CardsActionTypes.MoveMyActiveCardsWithinArray:
+      state.myActiveCards.splice(action.payload.currentIndex, 0, state.myActiveCards.splice(action.payload.previousIndex, 1)[0]);
```
Comment (vharadkou) [inline client/src/store/cards/cards.reducer.ts]: please copy all state first, 
`{
...state,
myActiveCards: state.myActiveCards.splice(...
}`

**Pull request merged**